org-static-blog.el

```diff
@@ -445,40 +455,51 @@ e.g. `(('foo' 'file1.org' 'file2.org') ('bar' 'file2.org'))`"
             (push (cons tag (list post-filename)) tag-tree)))))
     tag-tree))
 
+(defun org-static-blog--preview-region ()
+  "Find the start and end of the preview in the current buffer."
+  (goto-char (point-min))
+  (if org-static-blog-preview-end
+      (when (or (search-forward (or org-static-blog-preview-start "<p>") nil t)
+                (search-forward "<p>" nil t))
+        (let ((start (match-beginning 0)))
+          (or (search-forward org-static-blog-preview-end nil t)
+              (search-forward "</p>" nil t))
+          (cons start (point))))
+    (when (search-forward (or org-static-blog-preview-start "<p>") nil t)
+      (let ((start (match-beginning 0)))
+        (search-forward "</p>")
+        (cons start (point))))))
```

Comment:
It might simplify things to return `(buffer-substring-no-properties start (point))` here directly, instead of the two positions.